supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalJobExecutionFSM.scala

```diff
@@ -0,0 +1,248 @@
+package cromwell.backend.impl.local
+
+import java.nio.file.{FileSystems, Path, Paths}
+
+import akka.actor.FSM.NullFunction
+import akka.actor.{PoisonPill, Props, LoggingFSM}
+import com.typesafe.config.Config
+import cromwell.backend.BackendJobDescriptor
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionAbortedResponse, BackendJobExecutionFailedResponse, BackendJobExecutionResponse, BackendJobExecutionSucceededResponse}
+import cromwell.backend.BackendLifecycleActor.{BackendJobExecutionAbortFailedResponse, BackendJobExecutionAbortSucceededResponse, JobAbortResponse}
+import cromwell.backend.impl.local.LocalJobExecutionFSM._
+import cromwell.core.CallContext
+import org.slf4j.LoggerFactory
+import wdl4s._
+import wdl4s.values.{WdlFile, WdlValue}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Future, Promise}
+import scala.language.postfixOps
+import scala.sys.process._
+import scala.util.{Failure, Success, Try}
+
+object LocalJobExecutionFSM {
```

Comment:
This is an FSM, but to my broader point I'm not sure what the responsibilities of this class actually are in contrast with its parent.